src/directives/scaleselector.js

```diff
@@ -243,6 +247,7 @@ ngeo.ScaleselectorController.prototype.handleResolutionChange_ = function(e) {
  */
 ngeo.ScaleselectorController.prototype.handleViewChange_ = function(e) {
   this.registerResolutionChangeListener_();
+  this.handleResolutionChange_(null);
```

Comment:
Then is `registerResolutionChangeListener_` still usefull ? 
I don't really understand what it does, seems like it register/unregister on all view changes ..